<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/fs_main_window.qrc">
    <normaloff>:/icons/preferences</normaloff>:/icons/preferences</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="exitConfirmationCheckBox">
         <property name="text">
          <string>Exit confirmation</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="trayTab">
      <attribute name="title">
       <string>Tray</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QCheckBox" name="trayIconCheckBox">
         <property name="text">
          <string>Tray icon</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="allowMinimizeToTrayCheckBox">
         <property name="text">
          <string>Allow minimize to tray</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="showNotificationsCheckBox">
         <property name="text">
          <string>Show notifications</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Notification timeout:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="notificationTimeoutSpinBox">
         <property name="suffix">
          <string> s</string>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="cacheTab">
      <attribute name="title">
       <string>Cache</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Maximum items:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="cacheMaxItemsSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000000</number>
         </property>
         <property name="value">
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="useCacheCheckBox">
         <property name="text">
          <string>Use cache</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/fs_main_window.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trayIconCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>allowMinimizeToTrayCheckBox</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trayIconCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>allowMinimizeToTrayCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trayIconCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>showNotificationsCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trayIconCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>showNotificationsCheckBox</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showNotificationsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>notificationTimeoutSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useCacheCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>cacheMaxItemsSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
